xbmc/utils/Fanart.cpp

```diff
@@ -56,6 +55,21 @@ void CFanart::Pack()
   m_xml << fanart;
 }
 
+void CFanart::AddFanart(std::string image, std::string preview, std::string colors)
```

Comment:
Consider passing these std::string arguments by const reference to prevent unnecessary copies.